tools/mason/MasonUtils.chpl

```diff
@@ -101,14 +101,18 @@ proc runCommand(cmd, quiet=false) : string throws {
   try {
 
     var splitCmd = cmd.split();
-    var process = spawn(splitCmd, stdout=pipeStyle.pipe);
+    var process = spawn(splitCmd, stdout=pipeStyle.pipe, stderr=pipeStyle.pipe);
 
-    for line in process.stdout.lines() {
+    var line:string;
+    while process.stdout.readLine(line) {
       ret += line;
       if !quiet {
         write(line);
       }
     }
+    if !quiet {
+      while process.stderr.readLine(line) do write(line);
```

Comment:
Do we want to add stderr to `ret` as well?

Reply:
I don't think so. If you look at usages like:
```chpl
 380 private proc gitUrl() {
 381   var url = runCommand("git config --get remote.origin.url", true);
 382   return url;
 383 }
```
I think the stderr would mess that up, but I think that may be a bigger issue with the usage of `runCommand` in mason and whether or not we would want a warning to make usage like this fail. My vote would be to just add the error quieting for now and then possibly reconsider the `runCommand` usage at a later date.